templates/shared-memory-service.json

```diff
@@ -305,6 +318,59 @@
                         }
                      }
                   ],
+                  "initContainers": [
+                     {
+                        "name": "pem-to-keystore",
+                        "image": "registry.access.redhat.com/redhat-sso-7/sso71-openshift:1.1-16",
+                        "env": [
+                           {
+                              "name": "TLS_KEY_FILE",
+                              "value": "/var/run/secrets/openshift.io/serviceaccount/tls.key"
+                           },
+                           {
+                              "name": "CRT_FILE",
+                              "value": "/var/run/secrets/openshift.io/serviceaccount/tls.crt"
+                           },
+                           {
+                              "name": "KEYSTORE_PKCS12",
+                              "value": "/var/run/secrets/java.io/keystores/keystore.pkcs12"
+                           },
+                           {
+                              "name": "KEYSTORE_FILE",
+                              "value": "/var/run/secrets/java.io/keystores/${APPLICATION_NAME}.jks"
+                           },
+                           {
+                              "name": "KEYSTORE_PASSWORD",
+                              "value": "${KEYSTORE_PASSWORD}"
+                           }
+                        ],
+                        "command": [
+                           "/bin/bash"
+                        ],
+                        "args": [
+                           "-c",
+                           "rm -f $KEYSTORE_FILE && openssl pkcs12 -export -inkey $TLS_KEY_FILE -in $CRT_FILE -out $KEYSTORE_PKCS12 -password pass:$KEYSTORE_PASSWORD && keytool -importkeystore -noprompt -srckeystore $KEYSTORE_PKCS12 -srcstoretype pkcs12 -destkeystore $KEYSTORE_FILE -storepass $KEYSTORE_PASSWORD -srcstorepass $KEYSTORE_PASSWORD"
```

Comment:
I'm just wondering - don't we have OpenSSL in our OpenShift JDG image? If so, maybe we don't need dependency to RHSSO and we can convert pem to keystore while booting up our services?